frontend/src/components/Interactions/Edit.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <button
+        class="material-icons edit-button"
+        type="menu"
+        title="Toggle Menu"
+    >
+        more_vert
```

Comment:
Also, it jiggles when I click it for some reason

Reply:
Lel what.

Reply:
* only on firefox :/ it moves a few pixels right on mousedown and moves back on mouseup. idk why lel. dont worry about it imo